src/app/canvas/canvas.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { CanvasComponent } from './canvas/canvas.component';
+import { CardComponent } from './card/card.component';
+import { ArrowComponent } from './arrow/arrow.component';
+
+@NgModule({
```

Comment:
Out of curiosity, thoughts on Angular / TypeScript nowadays?

Would you rather see this app written in another tech like React?